app/models/concerns/scholars_archive/default_metadata.rb

```diff
@@ -184,6 +184,11 @@ module DefaultMetadata
         index.as :stored_searchable
       end
 
+      property :nested_geo_points, :predicate => ::RDF::URI("http://www.w3.org/2003/12/exif/ns#geo"), :class_name => NestedGeoPoint
```

Comment:
:nested_geo_points and :nested_geo_bbox can be combined into a single :nested_geo property. The predicate for that is https://purl.org/geojson/vocab#Feature